src/hooks/useLocalStorage.js

```diff
@@ -0,0 +1,9 @@
+import { useEffect } from 'react';
+
+const useLocalStorage = (key, state) => {
+  useEffect(() => {
+    localStorage.setItem(key, JSON.stringify(state));
```

Comment:
i think there should be a way to remove a way to remove an item from the localStorage or to clear it.